scripts/createdb

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+mysql -h mysql -u root -p${MYSQL_ROOT_PASSWORD} <<SQL_QUERY
+ CREATE USER IF NOT EXISTS 'ironic'@'localhost' IDENTIFIED BY '$MYSQL_PASSWORD' ;
+ CREATE USER IF NOT EXISTS 'ironic'@'%' IDENTIFIED BY '$MYSQL_PASSWORD' ;
+ GRANT ALL on *.* TO 'ironic'@'localhost' IDENTIFIED BY '$MYSQL_PASSWORD' WITH GRANT OPTION ;
```

Comment:
can remove this line.. also fix remove extra whitespaces in this file at line beginnings

Reply:
Fixed